vas2nets/tasks.py

```diff
@@ -0,0 +1,29 @@
+import csv
+import requests
+
+from django.conf import settings
+from celery.task import Task
+
+from .models import VoiceCall
+
+
+class FetchVoiceData(Task):
+
+    def get_data(self, date):
+        url = "%s?report_date=%s" % (settings.V2N_VOICE_URL, date)
+
+        content = requests.get(url).content.decode('utf-8')
```

Comment:
Maybe we should do a `stream=True` in the request, and then do a `.iter_lines()`, because it does seem like they're streaming the data on their side, and it will allow us to not have to load all the data into memory before processing it.